Form/EventListener/AddressEmbeddableTypeSubscriber.php

```diff
@@ -86,15 +86,8 @@ public function preSetData(FormEvent $event): void
 
         $form = $event->getForm();
 
-        foreach ($addressFormat->getGroupedFields() as $line_index => $line_fields) {
```

Comment:
getGroupedField() is no longer in the addressFormat, but is still there, in AddressFormatHelper::getGroupedFields() . That probably should be used to sort the fields correctly based on the country postal format.

Reply:
This still warrants some updating. I have a major change coming in a PR I'm gonna open up myself and let another dev review, where the logic of this subscriber is altered a bit to allow the default country to be set in here. I'd suggest reworking based off that if possible.